alibi_detect/utils/tensorflow/distance.py

```diff
@@ -83,7 +109,52 @@ def batch_compute_kernel_matrix(
     return k_mat
 
 
-def mmd2_from_kernel_matrix(kernel_mat: tf.Tensor, m: int, permute: bool = False,
+def linear_mmd2(x: tf.Tensor,
+                y: tf.Tensor,
+                kernel: Callable,
+                permute: bool = False) -> Tuple[tf.Tensor, tf.Tensor]:
```

Comment:
Is there a reason we offer permute option for tensorflow and not torch?

Reply:
Legacy issue, now removed for the tensorflow version.